decam_process.py

```diff
@@ -15,324 +15,378 @@
-            '--output', processedrepo,
-            '--calib', calibrepo,
-            '-C', OBS_DECAM_DIR + '/config/processCcdCpIsr.py',
-            '--config', 'calibrate.doAstrometry=True',
-            'calibrate.doPhotoCal=True']
-    ProcessCcdTask.parseAndRun(args=args, config=config)
+    if os.path.isdir(os.path.join(processedrepo, '0'+visit)):
+        print('ProcessCcd has already been run for visit {0}, skipping...'.format(visit))
+    else:
+        if not os.path.isdir(processedrepo):
+            os.mkdir(processedrepo)
+        print('Running ProcessCcd...')
+        OBS_DECAM_DIR = getPackageDir('obs_decam')
+        config = ProcessCcdConfig()
+        # Use gaia for astrometry (phot_g_mean_mag is only available DR1 filter)
+        # Use pan-starrs for photometry (grizy filters)
+        for refObjLoader in (config.calibrate.astromRefObjLoader,
+                             config.calibrate.photoRefObjLoader,
+                             config.charImage.refObjLoader,):
+            refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
+        config.calibrate.astromRefObjLoader.ref_dataset_name = 'gaia'
+        config.calibrate.astromRefObjLoader.filterMap = {'u': 'phot_g_mean_mag',
+                                                         'g': 'phot_g_mean_mag',
+                                                         'r': 'phot_g_mean_mag',
+                                                         'i': 'phot_g_mean_mag',
+                                                         'z': 'phot_g_mean_mag',
+                                                         'y': 'phot_g_mean_mag',
+                                                         'VR': 'phot_g_mean_mag'}
+        config.calibrate.photoRefObjLoader.ref_dataset_name = 'pan-starrs'
+        config.calibrate.photoRefObjLoader.filterMap = {'u': 'g',
+                                                        'g': 'g',
+                                                        'r': 'r',
+                                                        'i': 'i',
+                                                        'z': 'z',
+                                                        'y': 'y',
+                                                        'VR': 'g'}
+        args = [repo, '--id', 'visit=' + visit, 'ccdnum=' + ccdnum,
+                '--output', processedrepo,
+                '--calib', calibrepo,
+                '-C', OBS_DECAM_DIR + '/config/processCcdCpIsr.py',
+                '--config', 'calibrate.doAstrometry=True',
+                'calibrate.doPhotoCal=True']
+        ProcessCcdTask.parseAndRun(args=args, config=config)
     return
 
 
 def doDiffIm(processedrepo, sciencevisit, ccdnum, templatevisit, diffimrepo):
     '''
     Do difference imaging with a visit as a template and one or more as science
-
-    USAGE:
-    $ python hits_ingest.py diffIm
+    
+    Parameters
+    ----------
+    processedrepo: `str`
```

Comment:
I first read this as `proccess drepo`. I like camel-case or underscore (I don't care which but I think the style guide has us moving toward using underscore, e.g. `processed_repo`.